tests/test_06_l2vpn_return_codes.py

```diff
@@ -317,9 +312,9 @@ def test_052_create_l2vpn_with_max_number_oxps_out_of_range_code410(self):
             }
         }
         response = requests.post(api_url, json=payload)
-        assert response.status_code == 410, response.text
-
-    @pytest.mark.xfail(reason="return status 201: Connection published")
+        assert response.status_code == 400, response.text
+         
+    @pytest.mark.xfail(reason="return status 400 -> Validation error: Scheduling not possible: 2025-01-22 16:47:43.982545+00:00 start_time cannot be before the current time\nScheduling not possible: 2023-12-30T12:00:00Z end_time is not in a valid ISO format")
```

Comment:
Related to issue [#389](https://github.com/atlanticwave-sdx/sdx-controller/issues/389)